gef.py

```diff
@@ -5778,7 +5787,7 @@ def usage(self, meth=None):
             gef_print(self.sock.system.methodHelp(m))
         return
 
-    def import_structures(self, structs):
+    def import_structures(self, structs) -> None:
```

Comment:
isn't `structs` `List[str]`?

Reply:
I think it's `Dict[str, [Any, str, int]]` but I'm not 100% sure.

I conclude this from:
```py
for struct_name in structs:
    ....
```
and
```py
for _, name, size in structs[struct_name]:
    ...
```

Reply:
that makes sense.

Reply:
actually `structs` seems to be something like `Dict[str, Tuple[Any, ...]]` (its the return value from calls to `importstruct()` in `ida_gef.py`

Reply:
ok, so it's:

```py
Dict[str, List[Tuple[int, str, int]]]
```

The tuple is (offset, name, size):

https://www.hex-rays.com/products/ida/support/idapython_docs/idautils.html#idautils.StructMembers

Reply:
great thanks for figuring that out.